tests/php/src/Cli/Export/ExportOptions.php

```diff
@@ -0,0 +1,359 @@
+	 *
+	 * @param array        $options       Associative array of options to adapt.
+	 * @param ExportResult $export_result Export result to adapt.
+	 * @return array Adapted associative array of options.
+	 */
+	private function adapt_options( $options, $export_result ) {
+		foreach ( $options as $key => $value ) {
+			switch ( $key ) {
+				case 'woocommerce_shop_page_title':
+				case 'woocommerce_cart_page_title':
+				case 'woocommerce_checkout_page_title':
+				case 'woocommerce_myaccount_page_title':
+				case 'woocommerce_edit_address_page_title':
+				case 'woocommerce_view_order_page_title':
+				case 'woocommerce_change_password_page_title':
+				case 'woocommerce_logout_page_title':
+					// @TODO
+					break;
+
+				case 'page_for_posts':
+				case 'page_on_front':
+					$options[ $key ] = $this->get_page_title_from_post_id( $value );
+					break;
+
+				case 'nav_menu_locations':
+					$options[ $key ] = $this->get_nav_menu_location( $value );
+					break;
+
+				case 'woocommerce_product_cat':
+					// @TODO
+					break;
+
+				case 'custom_logo':
+					$media_uploader = new MediaFileUploader();
+					$options[ $key ] = $media_uploader->upload(
+						$export_result->get_site_name(),
+						$this->get_attachment_url_from_attachment_id( $value )
+					);
+					break;
+			}
+		}
+
+		return $options;
+	}
+
+	/**
+	 * Skip the options that are marked as excluded.
+	 *
+	 * @param string $option_key Option key to check.
+	 * @return bool Whether to keep the option.
+	 */
+	private function skip_excluded_options( $option_key ) {
+		foreach ( self::EXCLUDED_OPTIONS as $option_pattern ) {
+			if ( 1 === preg_match( $option_pattern, $option_key ) ) {
+				return false;
+			}
+		}
+
+		return true;
+	}
+
+	/**
+	 * Skip the options that still have their default value.
+	 *
+	 * @param string $option_value Option value to check.
+	 * @param string $option_key   Option key to check.
+	 * @return bool Whether to keep the option.
+	 */
+	private function skip_default_values( $option_value, $option_key ) {
+		if ( ! array_key_exists( $option_key, self::OPTION_DEFAULTS ) ) {
+			return true;
+		}
+
+		return self::OPTION_DEFAULTS[ $option_key ] !== $option_value;
+	}
+
+	/**
+	 * Get the post title from a post ID.
+	 *
+	 * @param string|int $post_id ID of the post to get the title from.
+	 * @return string Post title.
+	 */
+	private function get_page_title_from_post_id( $post_id ) {
+		$post = get_post( $post_id );
+
+		if ( ! $post instanceof WP_Post || empty( $post->post_title ) ) {
+			return $post_id;
+		}
+
+		return $post->post_title;
+	}
+
+	/**
+	 * Get the attachment URL from an attachent ID.
+	 *
+	 * @param string|int $attachment_id ID of the attachment post to get the URL
+	 *                                  for.
+	 * @return string|false Attachment URL, or false if none.
+	 */
+	private function get_attachment_url_from_attachment_id( $attachment_id ) {
+		return wp_get_attachment_url( $attachment_id );
+	}
```

Comment:
Why is this method needed? Seems like a verbose alias.

Reply:
For most of these "adaptation" methods, they started out as basically aliases but then grew further and further as I fixed bugs and edge cases. This one happens to be a WP function that is properly built, I guess?

Happy to remove this indirection, though.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/5162/commits/df2c3f8944bc577a623b299525c150022ba19f46